src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -1,15 +1,11 @@
-/* Blue duo-tone for the main region landing page image */
-$duo-tone-darken: #112e51;
-$duo-tone-lighten: #000;
```

Comment:
`#112e51` is `$color-primary-darkest`:

![image](https://user-images.githubusercontent.com/2008881/119413462-8199db80-bca2-11eb-8dbb-89f395027a66.png)


`#000` is `$color-black`

https://design.va.gov/design/color-palette